src/components/BentoBoxes/index.tsx

```diff
@@ -0,0 +1,204 @@
+      children:
+        "Stablecoins are currencies that maintain stable value. Their price matches the U.S. dollar or other steady asset",
+      action: "Learn more",
+      href: "/stablecoins/",
+      imgSrc: ImpactImage,
+      imgWidth: 500,
+      className: cn(
+        "border-primary/10",
+        positions,
+        colors("primary"),
+        flow.mobile.down,
+        flow.lg.up,
+        flow.xl.right,
+        "lg:col-span-6 lg:row-start-2",
+        "xl:col-span-7 xl:col-start-5 xl:row-start-1"
+      ),
+    },
+    {
+      title: "A fairer financial system",
+      children:
+        "Billions can't open bank accounts or freely use their money. Ethereum's financial system is always open and unbiased.",
+      action: "Explore DeFi",
+      href: "/defi/",
+      imgSrc: ManAndDogImage,
+      className: cn(
+        "border-accent-c/10",
+        positions,
+        colors("accent-c"),
+        flow.mobile.down,
+        flow.lg.down,
+        flow.xl.up,
+        "lg:col-span-6 lg:col-start-7 lg:row-start-2",
+        "xl:col-span-4 xl:col-start-2 xl:row-start-2"
+      ),
+    },
+    {
+      title: "Innovative apps",
+      children:
+        "Ethereum apps work without selling your data. Protect your privacy.",
+      action: "Browse apps",
+      href: "/dapps/",
+      imgSrc: MergeImage,
+      imgWidth: 320,
+      className: cn(
+        "border-accent-b/10",
+        positions,
+        colors("accent-b"),
+        flow.mobile.down,
+        flow.lg.right,
+        flow.xl.down,
+        "lg:col-span-12 lg:row-start-3",
+        "xl:col-span-3 xl:col-start-6 xl:row-start-2"
+      ),
+    },
+    {
+      title: "The network of networks",
+      children:
+        "Ethereum is the hub for blockchain innovation. The best project are built on Ethereum.",
+      action: "Explore benefits",
+      href: "/layer-2/",
+      imgSrc: ManBabyWomanImage,
+      className: cn(
+        "border-accent-a/10",
+        positions,
+        colors("accent-a"),
+        flow.mobile.down,
+        flow.lg.up,
+        flow.xl.up,
+        "lg:col-span-6 lg:col-start-7 lg:row-start-4",
+        "xl:col-span-3 xl:col-start-9 xl:row-span-2 xl:row-start-2"
+      ),
+    },
+    {
+      title: "The internet of assets",
+      children:
+        "Arts, certificates or even real estate can be tokenized. Anything can be a tradable token. Ownership is public and verifiable.",
+      action: "More on NFTs",
+      href: "/nft/",
+      imgSrc: RobotBarImage,
+      imgWidth: 324,
+      className: cn(
+        "border-primary/10",
+        positions,
+        colors("primary"),
+        flow.mobile.down,
+        flow.lg.down,
+        flow.xl.right,
+        "lg:col-span-6 lg:row-start-4",
+        "xl:col-span-7 xl:col-start-2 xl:row-start-3"
+      ),
+    },
+  ]
+
+  return (
+    <>
+      {/* Mobile */}
+      <div className="relative my-16 max-w-[min(calc(100vw_-_2rem),_30rem)] lg:hidden">
```

Comment:
If we want to make it reusable, the parent should control the margins.

```ts
// in order to do this
<BentoBoxes className="my-16">

// or
<BentoBoxes className="my-8 lg:my-16 w-1/2">
```

So, maybe you need to have a common parent 
```tsx
return (
  <div className={cn("relative", className)}>
    {/* Mobile */}
    <div className="lg:hidden">
    
    {/* Desktop */}
    <div className="hidden lg:block">
  </div>
)
```